test/run-drun/brittle-stable-functions/RenamedHack.mo

```diff
@@ -0,0 +1,18 @@
+persistent actor {
+  class Clamper(l : Nat, h : Nat) {
+    public func clamp(x : Nat) : Nat {
+      // When trying to evolve this class to hardcode the upper limit,
```

Comment:
That's odd, I'd expect the loss of a free variable to be tolerated. Maybe this a bug rather than design flaw